submissions/Vitaminvp/HTML&CSSPractice/style.css

```diff
@@ -0,0 +1,237 @@
+body {
+    font-family: "Arial", sans-serif;
+    background-color: #fff;
+    overflow-x: hidden;
+}
+ul {
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+}
+.nav{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.nav__input,
+.nav__menu,
+.nav__menu_input,
+.nav__menu_listMore{
+    display: none;
+}
+.nav__menu{
+    position: absolute;
+    top: 50px;
+    border: 1px solid #ccc;
+    border-color: rgba(0,0,0,.2);
+    box-shadow: 0 2px 10px rgba(0,0,0,.2);
+    background: #fff;
+    border-radius: 2px;
+    width: 337px;
+    max-height: 634px;
+    height: 496px;
+    overflow-y: overlay;
+    animation-name: google;
+    animation-duration: 0.5s;
+    animation-timing-function: ease;
+    animation-delay: 0s;
+    animation-iteration-count: 1;
+    animation-direction: normal;
+    animation-fill-mode: none;
+    animation-play-state: running;
+}
+.nav__menu_rect{
+    content: "";
+    border-color: transparent;
+    border-bottom-color: #fff;
+    border-style: dashed dashed solid;
+    border-width: 0 8.5px 8.5px;
+    box-shadow: 0 2px 10px rgba(0,0,0,.2);
+    position: absolute;
+    left: 6.5px;
+    top: -9px;
+    z-index: 1;
+    height: 0;
+    width: 0;
+}
+.nav__menu .nav__menu_list{
+    display: flex;
+    flex-wrap: wrap;
+    margin: 20px;
```

Comment:
You can also use `em` and `rem` units for styling elements. But this is up to you.